### PR TITLE
fix: adding semantic release script

### PR DESCRIPTION
**Pull request by davidjoy**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -59,18 +59,14 @@
     "email-prop-type": "1.1.7",
     "font-awesome": "4.7.0",
     "form-urlencoded": "3.0.0",
-    "glob": "7.1.3",
     "history": "4.7.2",
-    "i18n-iso-countries": "3.7.8",
-    "iso-countries-languages": "0.2.1",
```
Comment (robrap) [inline package.json]: This is meant to just go away?

File: package.json [hunk 2]
```diff
@@ -59,18 +59,14 @@
     "email-prop-type": "1.1.7",
     "font-awesome": "4.7.0",
     "form-urlencoded": "3.0.0",
-    "glob": "7.1.3",
```
Comment (robrap) [inline package.json]: This is a dev dependency?

File: package.json (hunk 1 above)
Comment (davidjoy) [reply package.json]: Yes.  They are unused - they got moved into frontend-i18n

File: package.json (hunk 2 above)
Comment (davidjoy) [reply package.json]: Yup.

**Pull request merged**